tsconfig.json

```diff
@@ -10,6 +10,13 @@
     "sourceMap": true,
     "outDir": "./dist",
     "baseUrl": "./",
+    "paths": {
+      "@app/*": ["src/*"],
+      "@utils/*": ["src/utils/*"],
+      "@constants/*": ["src/constants/*"],
+      "@entities/*": ["src/entities/*"],
```

Comment:
Probably it will be better to keep entity in the folder with the module, but it is up to you.